app/views/poems/show.html.erb

```diff
@@ -1,22 +1,32 @@
-<div class="page-header">
-  <h1><%= @poem.title %></h1>
+<div class="row">
+  <div class="col-xs-6 col-md-offset-3">
+    <div class="panel-body">
+      <div class="page-header">
+        <h1><%= @poem.title %></h1>
+      </div>
+      <div class="poem-area">
+        <p>
+          <%= markdown(@poem.description) %>
+        </p>
+      </div>
+    </div>
+  </div>
 </div>
-<div class="poem-area">
-  <p>
-    <%= markdown(@poem.description) %>
-  </p>
+<div class="row">
+  <div class="col-xs-6 col-md-offset-3">
+    <%= link_to "読んだよ", poem_read_poems_path(@poem), method: :post, class: "btn btn-primary"%>
+    <% @read_poems.each do |read_poem| %>
+      <span><%= link_to image_tag(read_poem.user.image_url, class: "img-circle", width: 35 ),
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)